samples/nrf9160/spm/README.rst

```diff
@@ -30,6 +30,17 @@ Requirements for the application firmware
 
 * The application firmware must be built as a non-secure firmware for the nrf9160_pca10090ns board.
 
+Automatic building of SPM
+=========================
+
+The sample is automatically built by non-secure applications when the nrf9160_pca10090ns board is used.
+
+However, you might want to flash SPM or the non-secure application individually.
+To do this, disable the automatic building of SPM by setting the option ``CONFIG_SPM=n`` in the ``prj.conf`` file of the application.
```

Comment:
Isn't this now CONFIG_IS_SPM? Maybe this was changed already. @hakonfam  to confirm.

Reply:
No, to enable-disable SPM from the "parent" image, the config called "CONFIG_SPM" is used.

To be built as the SPM subsystem, you use the "CONFIG_IS_SPM" config. 

So what is here is correct.